examples/asset/asset_dependency.rs

```diff
@@ -0,0 +1,136 @@
+/// This example demonstrates how to handle loading dependency assets.
```

Comment:
As far as I can see, most examples are documented by using the `//!` comment block, which basically means you are documenting the module itself. Maybe you could try to use it here too to keep consistency?

Reply:
Yep forgot //! existed 😅 Thanks!